Code/Tools/ProjectManager/Source/CreateAGemScreen.h

```diff
@@ -35,7 +35,7 @@ namespace O3DE::ProjectManager
         ~CreateGem() = default;
 
     signals:
-        void CreateButtonPressed();
+        void CreateButtonPressed(GemInfo gemInfo);
```

Comment:
Lets use a const ref to avoid copying the data more than needed
```suggestion
        void CreateButtonPressed(const GemInfo& gemInfo);
```